### PR TITLE
Merging master to codeReview

### PR DESCRIPTION
**Pull request by kwons90**
Yay

### CODE REVIEW

**Comment by kwons90** [comment]
Deanna: 'these commits are sub-par'

**Comment by kwons90** [comment]
Deanna: 'i have seen better commits'

File: client/components/nav.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        </Link>
+        <Link to="/cart" style={{ color: "white" }}>
+          Cart
+        </Link>
+      </nav>
+    </div>
+  );
+};
+
+// const mapStateToProps = (state) => {
```
Comment (deannawong) [inline client/components/nav.js:29]: Delete commented out code if it's not being used.

File: client/components/search.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  constructor() {
+    super();
+    this.state = {
+      searchTerm: "",
+    };
+  }
+
+
+  async componentDidMount(){
+      console.log("component mounted ");
```
Comment (deannawong) [inline client/components/search.js:16]: Delete console.log's that aren't useful anymore.

File: server/api/routers/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const apiRouter = require('express').Router();
```
Comment (deannawong) [inline server/api/routers/index.js:1]: yay routers!

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+// const express = require('express');
```
Comment (deannawong) [inline server/index.js:1]: What is this file for?

**Pull request merged**